docs/networking.md

```diff
@@ -459,20 +459,46 @@ $ kops create cluster \
 
 You can adjust Cilium agent configuration with most options that are available in [cilium-agent command reference](http://cilium.readthedocs.io/en/stable/cmdref/cilium-agent/).
 
-E.g enabling logstash integration would require you to change above block to
+The following command will launch your cluster with desired Cilium configuration
 
+```console
+$ kops update cluster myclustername.mydns.io --yes
 ```
+
+##### Enabling BPF NodePort
+
+As of Kops 1.18 you can safely enable Cilium NodePort.
+
+In this mode, the cluster is fully functional without kube-proxy, with Cilium replacing kube-proxy's NodePort implementation using BPF.
+Read more about this in the [Cilium docs](https://docs.cilium.io/en/stable/gettingstarted/nodeport/)
+
+Be aware that you need to use an AMI with at least Linux 4.19.57 for this feature to work.
+
+```
+  kubeProxy:
+    enabled: false
   networking:
     cilium:
-      logstash: true
+      enableNodePort: true
 ```
 
-The following command will create your cluster with desired Cilium configuration
+##### Enabling Cilium ENI IPAM
 
-```console
-$ kops update cluster myclustername.mydns.io --yes
+As of Kops 1.18, you can have Cilium provision AWS managed adresses and attach them directly to Pods much like Lyft VPC and AWS VPC. See [the Cilium docs for more information](https://docs.cilium.io/en/v1.6/concepts/ipam/eni/)
+
+When using ENI IPAM you need to disable masquerading in Cilium as well.
```

Comment:
```suggestion
When using ENI IPAM you need to disable masquerading in Cilium unless you have no NAT/internet gateway to masquerade the ENI addresses to external networks.
```

Reply:
The PR went in with forcing masquerading to be disabled.

Reply:
That seems unfortunate. I had thought we had discovered a use case for not forcing it.